tests/testthat/test-drive_cp.R

```diff
@@ -0,0 +1,75 @@
+context("Copy files")
```

Comment:
Previously, the `drive_cp()` tests weren't cleaning up after themselves. And due to use of `drive_search(pattern = ...)` vs `drive_path()` were creating a LOT of copies of copies of the test file(s). I deleted ~120 file just from this.